assembl/tests/model_tests/test_discussion.py

```diff
@@ -119,4 +120,3 @@ def test_get_roles_by_permission(
     roles_by_permission = discussion_with_permissions.get_roles_by_permission()
     assert simple_permission.name in roles_by_permission
     assert simple_role.name in roles_by_permission[simple_permission.name]
-
```

Comment:
please don't do this, it is not ok with this `.editorconfig` option : 
```insert_final_newline = true```

(use a editorconfig plugin for your editor to avoid this)